snapshot/lib/snapshot/test_command_generator.rb

```diff
@@ -29,6 +29,7 @@ def pipe(language: nil, locale: nil, log_path: nil)
       end
 
       def destination(devices)
+        puts("devices: #{devices.inspect}")
```

Comment:
This will be removed before merging, right?

Reply:
Yes yes ^